### ISSUE TITLE
Parental control and safe browsing options cause SERVFAIL in the responses and are not working 

### ISSUE THREAD

**Issue by Versty** [AdguardTeam/AdGuardHome#6181]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, AMD64 (aka x86_64)

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

v0.107.37

### Action

0. Install and configure AdGuard Home 
1. Enable parental control option 
2. Launch terminal and perform `nslookup xxnx.com` command 

### Expected result

The IP address of AdGuard's protection page is displayed in response 
The adult website is marked as blocked by parental control in the query log. 

### Actual result

The SERVFAIL error is displayed in the response section.  
The adult website is marked as blocked by parental control in the query log. 

### Additional information and/or screenshots

  `parental_block_host: ""` and `safebrowsing_block_host: ""` fields are empty in the default AdGuardHome.yaml config file.

**Comment by ainar-g** [comment]
Working on a fix.

The workaround is to:

1.  Stop AdGuard Home.

2.  Edit the configuration file, `AdGuardHome.yaml`, to have:

    ```yaml
    # …
    filtering:
      # …
      parental_block_host: 'family-block.dns.adguard.com'
      safebrowsing_block_host: 'standard-block.dns.adguard.com'
      # …
    ```

3.  Start AdGuard Home.

**Comment by ainar-g** [comment]
Should be fixed on Edge, going to be released with v0.107.38.

**Comment by djparus** [comment]
> Working on a fix.
> 
> The workaround is to:
> 
> 1. Stop AdGuard Home.
> 2. Edit the configuration file, `AdGuardHome.yaml`, to have:
>    ```yaml
>    # …
>    filtering:
>      # …
>      parental_block_host: 'family-block.dns.adguard.com'
>      safebrowsing_block_host: 'standard-block.dns.adguard.com'
>      # …
>    ```
> 3. Start AdGuard Home.

It's not work
